+turso — deploy your databases to the edge

kevinmichaelchen opened the pull request:
https://turso.tech/

kevinmichaelchen committed:
+turso

jhheider commented on projects/turso.tech/package.yml:
```diff
@@ -0,0 +1,37 @@
+
+build:
+  script: |
+    git clone https://github.com/tursodatabase/turso-cli
```
Instead of this, either you should make the `distributable.url: https://github.com/tursodatabase/turso-cli/archive/refs/tags/v{{version}}.tar.gz` (and use the same github for the version), or make the `distributable: { url: git+https://github.com/tursodatabase/turso-cli, ref: v{{version}} }`, if it _strictly requires_ being built from a git checkout (rare, but it exists).

kevinmichaelchen replied:
trying the former approach (see https://github.com/teaxyz/pantry/pull/2856/commits/7228003f2fed515c97857ba573edbbaad805d4a2).

```
no versions parsed. Re-run with DEBUG=1 to see output.
{ versions: [] }
error: Uncaught Error: not-found: version: turso.tech
```

Maybe it's a "Tags vs Releases" thing? The Turso repo [has Tags but not Releases](https://github.com/tursodatabase/turso-cli/tags).

jhheider replied:
Looks like setting the version depends on passing `-tags=prod` to the build command.

kevinmichaelchen committed:
fix: version

https://github.com/tursodatabase/turso-cli/blob/v0.80.0/internal/cmd/version_prod.go\#L12

kevinmichaelchen committed:
wip

jhheider committed:
check tags for version

jhheider committed:
fix version logic

jhheider commented:
If you're happy with the above fixes, looks like this is ready, @kevinmichaelchen .

kevinmichaelchen commented:
@jhheider Thanks for the fixes! LGTM.
